### DISCUSSION TITLE
Access volumes through proxy

### DISCUSSION THREAD

**Discussion by facorazza** [seaweedfs/seaweedfs-csi-driver#158] [Q&A] [answered]
The Seaweed cluster I'm trying to use is external to the k8s cluster and I'm getting connection timeout in the csi-seaweedfs-plugin.
How can I define the `volumeServerAccess=filerProxy` argument using the helm chart?

**Comment by chrislusf** [comment, marked as answer]
this is not built yet https://github.com/seaweedfs/seaweedfs-csi-driver/blob/master/pkg/driver/mounter_seaweedfs.go#L86

**Reply by facorazza** [reply]
I see thanks

**Comment by TheHackmeister** [comment]
It's definitely worth building into the csi driver, but a usable workaround until then is to have CoreDNS rewrite the internal request to point externally like this: https://github.com/OwnYourIO/SpencersLab/blob/ae68426d78892b0d89942c8ec43374cd35af42ba/services/media/prod/templates/pvc-media-seaweedfs.yaml#L38-L51

**Reply by chrislusf** [reply]
@TheHackmeister Good idea! Or send a PR to add the parameter?
